gemini-app/src/Components/GCP/ImageViewer.js

```diff
@@ -77,127 +178,145 @@ function ImageViewer({ open, onClose, item, activeTab, platform, sensor }) {
-                            <Button variant="contained" color="warning" onClick={() => setOrthoModalOpen(true)}>Generate Orthophoto</Button>
+                            <Slider
+                                value={imageIndex}
+                                onChange={(event, newValue) => setImageIndex(newValue)}
+                                aria-labelledby="image-slider"
+                                step={1}
+                                marks={sliderMarks}
+                                min={0}
+                                max={imageList.length - 1}
+                                valueLabelDisplay="auto"
+                                valueLabelFormat={(value) => `${value + 1} of ${imageList.length}`}
+                                track={false}
+                                sx={{
+                                    width: '80%', // Adjust this value to control the slider's width
+                                    "& .MuiSlider-rail": {
+                                        height: SLIDER_RAIL_HEIGHT,
+                                    },
+                                    "& .MuiSlider-thumb": {
+                                        width: SLIDER_THUMB_SIZE,
+                                        height: SLIDER_THUMB_SIZE,
+                                    },
+                                }}
+                            />
+                            <div style={{
+                                display: 'flex',
+                                justifyContent: 'space-around', // Evenly spaces items along the line
+                                // width: '80%', // Match the slider width for alignment,
+                                gap: '20px', // Adds space between the buttons
+                            }}>
+                                <Button variant="contained" onClick={handlePrevious}>Previous</Button>
+                                <Button variant="contained" onClick={handleNext}>Next</Button>
+                                <Button variant="contained" color="warning" onClick={() => setOrthoModalOpen(true)}>Generate Orthophoto</Button>
+                            </div>
                         </div>
-                    </div>
-                )}
-                <OrthoModal />
-            </div>
-        </Dialog>
+                    )}
+                    <OrthoModal />
+                </div>
+            </Dialog>
+            <Dialog open={dialogOpen} onClose={handleDialogClose}>
+                <DialogTitle>Upload GCP Locations</DialogTitle>
+                <DialogContent>
+                    <DialogContentText>
+                        Please upload the gcp_locations.csv file before proceeding for maximum orthomosaic quality.
+                        For more information, press <a href="https://gemini-breeding.github.io/1.%20App/2-%20File%20Upload/" target="_blank" rel="noopener noreferrer">here</a>.
```

Comment:
Link to info page.
<img width="492" alt="Screenshot 2024-12-17 at 3 18 36 PM" src="https://github.com/user-attachments/assets/5951a05e-ec35-4135-a8b5-bd6a56dc8d1c" />